_includes/charts/calico/templates/calico-kube-controllers.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or (eq .Values.datastore "etcd") (eq .Values.network "calico") }}
+{{- if or (eq .Values.datastore "etcd") (eq .Values.network "calico") (eq .Values.network "flannel") }}
```

Comment:
I think with this change we're saying that we run kube-controllers all the time, right? Do we need to limit to "flannel" || "calico" || "etcd"?

Also, what is the scenario where you're not using flannel, nor calico, but you are using etcd?

Reply:
Thanks, fixed